R/meta.R

```diff
@@ -0,0 +1,39 @@
+
+.roxygen_meta <- new.env(parent = emptyenv())
+
+roxy_meta_read <- function(name) {
+  .roxygen_meta[[name]]
+}
+
+roxy_meta_load <- function(base_path) {
+
+  meta_dir <- file.path(base_path, "man-roxygen")
+
+  pattern <- "^roxygen-meta[.][Rr]"
```

Comment:
It seems weird to use `roxygen` twice here. How about `man/roxygen/options.R`? Can you check if R CMD check warns about such a file/directory? (i.e. do we need to build-ignore it too?)